examples/gradle-pipeline/README.md

```diff
@@ -37,8 +37,8 @@ fi
 crane ls "$(echo -n ${IMAGE_URL} | sed 's|:[^/]*$||')"
 
 # Verify the image and the attestation.
-cosign verify --key k8s://tekton-chains/signing-secrets "${IMAGE_URL}"
-cosign verify-attestation --type slsaprovenance --key k8s://tekton-chains/signing-secrets "${IMAGE_URL}"
+cosign verify --insecure-ignore-tlog --key k8s://tekton-chains/signing-secrets "${IMAGE_URL}"
```

Comment:
`--insecure-ignore-tlog` is there a secure way to do this that we should be doing instead?

Reply:
Only if there is a record of the signing in Rekor. At least that's my understanding. If you want to verify "off-line" you have to use that option.

Reply:
If we switched to keyless signing then we could drop the `--insecure-ignore-tlog`.